bip-0003.md

```diff
@@ -0,0 +1,753 @@
+repository](https://github.com/bitcoin/bips) where it may get further feedback.
+
+For each new BIP pull request that comes in, an editor checks the following:
+
+* The idea has been previously discussed on the Bitcoin Development Mailing List
+* Title accurately describes the content
+* Proposal is of general interest and/or pertains to more than one Bitcoin project/implementation
+* Document is properly formatted
+* Licensing terms are acceptable
+* Motivation, Rationale, and Backward Compatibility have been addressed
+* Specification provides sufficient detail for implementation
+* The defined Layer header must be correctly assigned for the given specification
+* The BIP is ready: it is comprehensible, technically feasible, and all aspects are addressed as necessary
+
+Editors do NOT evaluate whether the proposal is likely to be adopted.
+
+A BIP editor will:
+
+* Assign a BIP number and BIP type in the pull request
+* Ensure that the BIP is listed in the [README](README.mediawiki)
+* Merge the pull request when it is ready
+
+The BIP editors are intended to fulfill administrative and editorial responsibilities. The BIP editors monitor BIP
+changes, and update BIP headers as appropriate.
+
+BIP editors may also, at their option, unilaterally make and merge strictly editorial changes to BIPs, such as
+correcting misspellings, fixing broken links, etc. as long as they do not change the meaning or conflict with the
+original intent of the authors. Such a change must be recorded in the Changelog if it’s noteworthy per the criteria
+mentioned in the [Changelog](#changelog) section.
+
+## Backward Compatibility
+
+### Changes from BIP 2
+
+#### Workflow
+
+- Status field values are reduced from nine to four:
+  - Deferred, Obsolete, Rejected, Replaced, and Withdrawn are gathered up into Closed.[^closed]
+  - Final and Active are collapsed into Deployed.
+  - Proposed is renamed to Complete.
+  - The remaining statuses are Draft, Complete, Deployed, and Closed.
+- The comment system is abolished.[^comments]
+- A BIP in Draft or Complete status may no longer be closed solely on grounds of not making progress for three years.[^rejection]
+  - A BIP in Draft status may be set to Closed by anyone if it appears to have stopped making progress for at least a
+    year and its authors do not assert that they are still working on it when contacted.
+  - Complete BIPs can only be moved to Closed by its authors and may remain in Complete indefinitely.
+- Process BIPs are living documents that do not ossify and may be modified indefinitely.
+- Some judgment calls previously required from BIP Editors are reassigned either to the BIP authors or the repository’s
+  audience.
+
+#### BIP Format
+
+- The Standards Track type is superseded by the similar Specification type.[^standard-track]
+- Many sections are declared optional, it is up to the authors and audience to judge whether all relevant topics have
```

Comment:
```suggestion
- Many sections are declared optional; it is up to the authors and reviewers to judge whether all relevant optional topics have
```

Reply:
I took the semicolon and change to "reviewers, but the additional "optional" seems unnecessary: all relevant topics have to be addressed sufficiently, regardless whether the sections are optional or not.